Overwrite native commands (#4049)
webdriverio/webdriverio: docs/CustomCommands.md

```

Note that the result of your custom command will be the result of the promise you return. Also there is no support for synchronous commands in standalone mode therefore you always have to handle asynchronous commands using promises.

### Overwriting native commands

> You can also overwrite native commands with `overwriteCommand`. It is highly unrecommended to do that because it may lead to unpredictable behavior of framework! Overall approach is similar to `addCommand`, the only difference is that first argument in command function is original function, please see some examples below.

Reviewer: ```suggestion
You can also overwrite native commands with `overwriteCommand`. It is not recommended to do this because it may lead to unpredictable behavior of the framework! The overall approach is similar to `addCommand`, the only difference is that the first argument in the command function is the original function that you are about to overwrite. Please see some examples below:
```

NOTE: examples below assumes sync mode. If you are not using it don't forget to add async/await.

#### Overwriting browser commands

```js
/**
 * print milliseconds before pause and return its value.
 */
// "pause"            - command to be overwritten
// origPauseFunction  - original pause function

> browser.overwriteCommand("pause", function (origPauseFunction, ms) {

Reviewer: ```suggestion
browser.overwriteCommand('pause', function (origPauseFunction, ms) {
```

    console.log(`sleeping for ${ms}`);
    origPauseFunction(ms);
    return ms;
});

// then use it as before
console.log(`was sleeping for ${browser.pause(1000)}`);
```

#### Overwriting element commands

Overwriting commands on element level is almost the same, simply pass `true` as 3rd argument to `overwriteCommand`

```js
/**
 * Attempt to scroll to element if it is not clickable.
 * Pass { force: true } to click with JS even if element is not visible or clickable.
 */
// "click"            - command to be overwritten
// origClickFunction  - original click function

> browser.overwriteCommand("click", function (origClickFunction, { force = false } = {}) {

Author: @CrispusDH take a look

Reviewer 2: thanks for example 👍

> browser.overwriteCommand("click", function (origClickFunction, { force = false } = {}) {

Reviewer: ```suggestion
browser.overwriteCommand('click', function (origClickFunction, { force = false } = {}) {
```
